ecs/user_data.sh

```diff
@@ -2,3 +2,6 @@
 
 echo "***** Connect to Cluster *****"
 echo ECS_CLUSTER=${ECS_CLUSTER} >> /etc/ecs/ecs.config
+
+echo "***** CloudWatch Logging *****"
+service awslogs start
```

Comment:
move to machine on start

Reply:
This can not be moved to the AMI, since it depends on being configured 
 with the INSTANCE_ID - https://github.com/tesera/terraform-modules/pull/107/files#diff-fd1e87e974c995c21e0c788899e4699fR17.